### PR TITLE
Droth 1187 railroad service road to be vizualised according to the huoltovastuu field

### PR DESCRIPTION
**Pull request by oskarmalmstrom**

### CODE REVIEW

File: UI/src/assetTypeConfiguration.js [hunk 1]
```diff
@@ -478,7 +478,7 @@
           {'name': "Tarkistettu", 'propType': 'checkbox', 'id': "huoltotie_tarkistettu", value: [{typeId: 0, title: 'Ei tarkistettu'}, {typeId: 1, title: 'Tarkistettu'}]}],
         style: new ServiceRoadStyle(),
         label : new ServiceRoadLabel(),
-        isVerifiable: true
+        isVerifiable: false
```
Comment (joanacoelho) [inline UI/src/assetTypeConfiguration.js]: It's suppose to change this, or was a mistake when testing?

File: UI/src/view/linear_asset/serviceRoadLayer.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+(function(root) {
+    root.ServiceRoadLayer  = function(params) {
+        LinearAssetLayer.call(this, params);
+        var me = this;
+        var style = params.style,
+            collection = params.collection;
+
+        var isResponsibilityStyle = true;
```
Comment (joanacoelho) [inline UI/src/view/linear_asset/serviceRoadLayer.js]: Can we change this name for something more clear?

File: UI/src/assetTypeConfiguration.js (hunk 1 above)
Comment (oskarmalmstrom) [reply UI/src/assetTypeConfiguration.js]: no, even though the asset is verifiable it was not included in the user story for the verification button. so the button did nothing before.

File: UI/src/application.js [hunk 3]
```diff
@@ -280,6 +280,31 @@
     };
 
     var linearAssetLayers = _.reduce(linearAssets, function(acc, asset) {
+      if(asset.layerName == 'maintenanceRoad') {
+        acc[asset.layerName] = new ServiceRoadLayer({
```
Comment (leirasf) [inline UI/src/application.js]: The layer could be setted in the asset type configuration

**Pull request merged**